platform/core/src/hooks/useToolbar.tsx

```diff
@@ -0,0 +1,59 @@
+import { useCallback, useEffect, useState } from 'react';
+
+export function useToolbar({ servicesManager, buttonSection = 'primary' }) {
```

Comment:
Docs please - also, if not already in the toolbar markdown docs, could you add the useToolbar documentation there?  This is a pretty important part of the GUI functionality.

Reply:
Oh, and I really like having this here.